packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -89,21 +114,202 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
     ): Promise<void> {
         if (buyNowInitializeOptions && typeof buyNowInitializeOptions.getBuyNowCartRequestBody === 'function') {
             const cartRequestBody = buyNowInitializeOptions.getBuyNowCartRequestBody();
-
             if (!cartRequestBody) {
                 throw new MissingDataError(MissingDataErrorType.MissingCart);
             }
 
             try {
                 const { body: cart } = await this._cartRequestSender.createBuyNowCart(cartRequestBody);
-
                 this._buyNowCartId = cart.id;
             } catch (error) {
                 throw new BuyNowCartCreationError();
             }
         }
     }
 
+
+    private async _onCancel() {
+        const state = this._store.getState();
+        const billingAddress = state.billingAddress.getBillingAddressOrThrow();
+
+        if (billingAddress) {
```

Comment:
BillingAddress check can be skipped due to line 133.

Reply:
removed